CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.2.8] - 2025-04-17
```

Comment:
Probably would still go for minor. The changes are not purely Bugfixes in my opinion.